File: Beecrowd/1794 Lavanderia/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n, la, lb, sa, sb;

    cin >> n;
    cin >> la >> lb;
    cin >> sa >> sb;

    if((n >= la and n <= lb) and (n >= sa and n <= sb))
        cout << "possivel" << endl;
    else
        cout << "impossivel" << endl;

    return 0;
}

File: Beecrowd/1794 Lavanderia/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, la, lb, sa, sb;

        n = sc.nextInt();
        la = sc.nextInt();
        lb = sc.nextInt();
        sa = sc.nextInt();
        sb = sc.nextInt();

        if ((n >= la && n <= lb) && (n >= sa && n <= sb))
            System.out.println("possivel");
        else
            System.out.println("impossivel");

        sc.close();

    }

}